README.md

```diff
@@ -83,4 +94,4 @@ We support the following `vis` commands:
 
 `:plug-checkout` - checkout {name} {branch|commit}
 
-`:plug-commands` - list all commands (these)
\ No newline at end of file
+`:plug-commands` - list all commands (these)
```

Comment:
Empty space? Can you remove this?

Reply:
The line was detected by the diff because of No new line at end of file

Reply:
Ok, just leave it